script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -74,11 +74,11 @@ It was a turning point in my entire life.
 It was the beginning of a new life for me, and the end of the old life of Tohno Shiki.
 <Page21>
 The scenery outside is changing little by little.
-We must have left the suburbs of Yashirogi which was separated by the river, and entered the urban area of Souya.
+We must have passed through Yashirogi, which was a suburb across the river, and entered the urban area of Souya.
```

Comment:
"which _was_ a suburb across the river" -> "which _is_ a suburb across the river? Yashirogi still exists

Reply:
Or elide it, and just have "Yashirogi, a suburb across the river,"